### ISSUE TITLE
Add /set_status command to change bot status

### ISSUE THREAD

**Issue by tjhubz** [tjhubz/PittBOT#36]
Throughout the year it might be fun to have the bot's status change. If there could be a /set_status (status) command to change the bot's status to a custom message it would be super cool.

[https://python.plainenglish.io/how-to-change-discord-bot-status-with-discord-py-39219c8fceea](https://python.plainenglish.io/how-to-change-discord-bot-status-with-discord-py-39219c8fceea)

**Comment by vrr11** [comment]
Should any arbitrary user be allowed to invoke this command? Or only RAs/managers?

**Comment by tjhubz** [comment]
> Should any arbitrary user be allowed to invoke this command? Or only RAs/managers?

Only users with administrator permissions.

**Comment by vrr11** [comment]
According to the webpage you linked: `Another limitation of the Discord API is that it does not at this time support custom statuses. This means that you can’t create a “Dreaming” activity for your bot to [dream of electric sheep](https://www.goodreads.com/book/show/36402034-do-androids-dream-of-electric-sheep).`

In other words, bots can currently only have statuses belonging to one of the following four categories:
![1_nWIV2NFTkQ6XRuWibNLzMg](https://user-images.githubusercontent.com/84156723/186295784-691efd38-44cf-4958-942d-f30b5d195bf5.png)
 


Do you still want this feature implemented?

**Comment by tjhubz** [comment]
Yep! Format should be /set_status (type) (time limit [1m, 1h, 1d, 1w etc.]) (message)
